### DISCUSSION TITLE
favicon branding in 0.73.0

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#6165] [General] [answered]
Hello

Following one of the Internet rules, telling there is always an xkcd comic about a problem one faces  (in this case https://xkcd.com/1172/), the bugfix 
> "The correct favicon is now used on login screen when public branding is enabled"

provided in 0.73.0 is confusing some of our users.

Our corporate site and outline have now the exact same favicon so people can't quickly find the outline tab at their web browser tabs (I'm part of them :)

is there a switch to remove this and see the wonderful outline logo in favicon :)

**Comment by tommoor** [comment, marked as answer]
Classic XKCD.

This was a bug fix in that the favicon should already have been the same throughout the rest of the app, it was just missing on the login screen. There are probably two options here –

- Disable "Public branding" in **Settings** -> **Details** which will remove the workspace logo from logged out pages.
- Update your Outline workspace logo to be some variant, perhaps a different color or inverted for example.
